### PR TITLE
chore: update linting

### PR DESCRIPTION
**Pull request by turadg**
Thought I'd need it for:
- https://github.com/Agoric/agoric-sdk/pull/5803
- https://github.com/Agoric/agoric-sdk/pull/5802

Turns out I don't but it's still good to have.

### CODE REVIEW

File: loadgen/types.js [hunk 1]
```diff
@@ -20,8 +20,6 @@ export {};
 /** @typedef {ReturnType<typeof import('@agoric/spawner').makeSpawner>} Spawner */
 /** @typedef {WalletUser & { getAdminFacet: () => WalletAdminFacet }} HomeWallet */
 
-/** @typedef { import('@agoric/run-protocol/src/vaultFactory/vaultFactory').VaultFactoryContract['publicFacet']} VaultFactoryPublicFacet */
-/** @typedef { import('@agoric/run-protocol/src/vaultFactory/vaultManager').CollateralManager} VaultCollateralManager */
```
Comment (mhofman) [inline loadgen/types.js:24]: Why remove this? I'd like to avoid relying on ambient types.

Comment (turadg) [reply loadgen/types.js:24]: > avoid relying on ambient types.

Agreed.

> Why remove this?

Deep imports are worse than ambient types. Moreover these paths are broken.

Comment (turadg) [reply loadgen/types.js]: Removed

**Review by mhofman — APPROVED** [review]
LGTM

**Pull request merged**